test/Interop/Cxx/value-witness-table/copy-constructor-irgen.swift

```diff
@@ -0,0 +1,45 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir | %FileCheck %s
+
+// REQUIRES: CPU=x86_64
+// REQUIRES: objc_interop
```

Comment:
Even without an `int` member, they need the stdlib to represent the reference types (with an `UnsafePointer`). I tried updating this test for multiple targets but, I don't think this will work (without the stdlib). I even tried mocking up a basic `UnsafePointer` in the test file but, that didn't seem to work either (and it seems very fragile). 

Another possibility is I could duplicate this test and have a `REQUIRES` comment for each target (i.e., have three tests for three targets). WDTY the best solution is?